manual/manual/tutorials/coreexamples.etex

```diff
@@ -110,11 +111,16 @@ sort [3.14; 2.718];;
 The "sort" function above does not modify its input list: it builds
 and returns a new list containing the same elements as the input list,
 in ascending order. There is actually no way in OCaml to modify
-in-place a list once it is built: we say that lists are  {\em immutable}
+a list in place once it is built: we say that lists are  {\em immutable}
```

Comment:
This should still be *in-place* with a hyphen, though the ordering is of course better.

Reply:
"in place" is the common English usage, not "in-place", right?  I moved the word and made this improvement.

Reply:
It's not common English, though - it's a technical term. cf. https://en.wikipedia.org/wiki/In-place_algorithm

Reply:
OK